src/main/java/frc/robot/RobotContainer.java

```diff
@@ -181,6 +182,24 @@ private void configureButtonBindings() {
         }));
     }
 
+    public BaseSwerveSubsystem getSwerveSubsystem(){
+        if (driveSubsystem instanceof BaseSwerveSubsystem){
+            return (BaseSwerveSubsystem) driveSubsystem;
+        }
+
+        else{
+            return null;
```

Comment:
This should probably throw a runtime exception here instead of passing in null and crashing anyways.